Keas.Mvc/ClientApp/components/Access/AccessEditValues.tsx

```diff
@@ -7,100 +8,97 @@ import SearchTags from '../Tags/SearchTags';
 interface IProps {
   selectedAccess: IAccess;
   disableEditing: boolean;
-  openEditModal?: (access: IAccess) => void;
   tags?: string[];
   error?: IValidationError;
+  children?: JSX.Element;
+  openEditModal?: (access: IAccess) => void;
   onAccessUpdate?: (access: IAccess) => void;
 }
 
-export default class AccessEditValues extends React.Component<
-  React.PropsWithChildren<IProps>
-> {
-  constructor(props: React.PropsWithChildren<IProps>) {
-    super(props);
+const AccessEditValues = (props: IProps) => {
+  useEffect(() => {
```

Comment:
What's the point of useEffect here?  If this is just running during every render you don't need the useEffect, and if it's just supposed to run once you need to add the second param

Reply:
Oh shootI thought it was used in a `componentDidMount` when it was really in a constructor. I will remove the useEffect